README-es.md

```diff
@@ -158,62 +158,62 @@ Notas:
 
 ## Procesamiento de archivos y datos
 
-- Para localizar un archivo por nombre en el directorio actual, `find . -iname '*something*'` (o similar). Para encontrar un archivo en cualquier lado por nombre, usar `locate something` (pero tenga en mente `updatedb` quizás no haya indexado recientemente los archivos creados).
+- Para localizar un archivo por nombre en el directorio actual, `find . -iname '*something*'` (o similar). Para encontrar un archivo en cualquier lado por nombre, use `locate something` (pero tenga en mente que `updatedb` quizás no haya indexado recientemente los archivos creados).
 
-- Por lo general buscar por la fuente o archivos de datos (más avanzado que `grep -r`), usar [`ag`](https://github.com/ggreer/the_silver_searcher).
+- Para busqueda general sobre la fuente o archivos de datos (más avanzado que `grep -r`), use [`ag`](https://github.com/ggreer/the_silver_searcher).
 
-- Para convertir HTML a texto: `lynx -dump -stdin`
+- Para convertir HTML a texto: `lynx -dump -stdin`.
 
-- Para Markdown, HTML, y todos los tipos de conversión de documentos, probar [`pandoc`](http://pandoc.org/).
+- Para Markdown, HTML, y todos los tipos de conversión de documentos, pruebe [`pandoc`](http://pandoc.org/).
 
-- Si debes manipular XML, `xmlstarlet` es viejo pero bueno.
+- Si debe manipular XML, `xmlstarlet` es viejo pero bueno.
 
-- Para JSON, usa [`jq`](http://stedolan.github.io/jq/).
+- Para JSON, use [`jq`](http://stedolan.github.io/jq/).
 
 - Para archivos Excel o CSV, [csvkit](https://github.com/onyxfish/csvkit) provee `in2csv`, `csvcut`, `csvjoin`, `csvgrep`, etc.
 
 - Para Amazon S3, [`s3cmd`](https://github.com/s3tools/s3cmd) es conveniente y [`s4cmd`](https://github.com/bloomreach/s4cmd) es el mas rápido. Hecho por Amazon [`aws`](https://github.com/aws/aws-cli) es esencial para otras tareas relacionadas al AWS.
 
-- Conocer acerca `sort` y `uniq`, incluyendo opciones de uniq `-u` y `-d` -- ver unas líneas más abajo.
+- Conocer acerca de `sort` y `uniq`, incluyendo las opciones de uniq `-u` y `-d` -- ver one-liners más abajo.
```

Comment:
Change:
 `one-liners` => `[one-liners](https://github.com/jlevy/the-art-of-command-line/blob/master/README-es.md#one-liners)`